packages/react-components/react-dialog/src/components/DialogActions/useDialogActions.ts

```diff
@@ -17,13 +17,13 @@ export const useDialogActions_unstable = (
 ): DialogActionsState => {
   const { position = 'end', fluid = false } = props;
   return {
-    components: {
-      root: 'div',
-    },
-    root: getNativeElementProps('div', {
-      ref,
-      ...props,
-    }),
+    root: slot<DialogActionsProps>(
+      getNativeElementProps('div', {
+        ref,
+        ...props,
+      }),
+      { required: true, componentType: 'div' },
```

Comment:
```suggestion
      { componentType: 'div' },
```

Is there sense to use `required`? It will be always defined anyway, correct?